spec/components/trainees/filters/view_spec.rb

```diff
@@ -19,19 +19,6 @@
     end
   end
 
-  context "when checkboxes have been pre-selected" do
```

Comment:
Hmmm if we remove this, we're not testing anywhere that checkboxes 'remember' whether they've been checked after you submit the form.

Instead of deleting, can we change the test to filter on status instead? Something like:

`let(:filters) { { state: %w[draft] }.with_indifferent_access }`

And then check `result.css("#state-draft")` is checked within the expectation.

Reply:
@AbigailMcP the spec below `when a subject has been pre-selected` should cover this case, no?

Reply:
@dankmitchell sadly the 'subject' is a select dropdown input. The code for pre-selecting that is different to pre-checking the checkbox inputs i.e. we're not testing the `def checked?` in component anymore.

I think we should have one test for every filter 'type'.

Reply:
Paired on this. All sorted